code/src/index.css

```diff
@@ -5,9 +5,180 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(0, 0, 0);
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  color:white;
+  margin-left: 5%;
+  margin-right: 5%;
+  border-bottom: 1px solid #65686e;
+  font-size: 27px;
+}
```

Comment:
We noticed that the h1 is not aligned with the body (the display of all the albums) as it is in the design. Perhaps this could be solved with the use of an outer and inner container for the entire site 🥳